css/css-flexbox/nested-flexbox-max-height.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<link rel="author" title="Przemyslaw Gorszkowski" href="mailto:pgorszkowski@igalia.com">
+<link rel="help" href="https://bugs.webkit.org/show_bug.cgi?id=282036">
```

Comment:
You can add a `<meta name="assert"` saying that the nested flex container needs to shrink to be 100px tall, and its item needs to be centered within these 100px.